lib/ansible/modules/network/ios/ios_static_routes.py

```diff
@@ -0,0 +1,508 @@
+        - NOTE, In case of IPV4/IPV6 VRF routing table should pre-exist before
+          configuring.
+        - NOTE, if the vrf information is not provided then the routes shall be
+          configured under global vrf.
+        type: str
+      address_families:
+        elements: dict
+        description:
+        - Address family to use for the static routes
+        type: list
+        suboptions:
+          afi:
+            description:
+            - Top level address family indicator.
+            type: str
+            required: true
+            choices:
+              - ipv4
+              - ipv6
+          routes:
+            description: Configuring static route
+            type: list
+            elements: dict
+            suboptions:
+              dest:
+                description: Destination prefix with its subnet mask
+                type: str
+                required: true
+              topology:
+                description:
+                - Configure static route for a Topology Routing/Forwarding instance
+                - NOTE, VRF and Topology can be used together only with Multicast and
+                  Topology should pre-exist before it can be used
+                type: str
+              next_hops:
+                description:
+                - next hop address or interface
+                type: list
+                elements: dict
+                suboptions:
+                  forward_router_address:
+                    description: Forwarding router's address
+                    type: str
+                  interface:
+                    description: Interface for directly connected static routes
+                    type: str
+                  dhcp:
+                    description: Default gateway obtained from DHCP
+                    type: bool
+                  distance_metric:
+                    description: Distance metric for this route
+                    type: int
+                  global:
+                    description: Next hop address is global
+                    type: bool
+                  name:
+                    description: Specify name of the next hop
+                    type: str
+                  multicast:
+                    description: multicast route
+                    type: bool
+                  permanent:
+                    description: permanent route
+                    type: bool
+                  tag:
+                    description:
+                    - Set tag for this route
+                    - Refer to vendor documentation for valid values.
+                    type: int
+                  track:
+                    description:
+                    - Install route depending on tracked item with tracked object number.
+                    - Tracking does not support multicast
+                    - Refer to vendor documentation for valid values.
+                    type: int
+  state:
```

Comment:
Since the states behave in the same manner, IMO, its redundant to describe it in every single RM. I have opened a docs request to add the description of the network resource module states in one place and then add a link to this doc in every RM.

https://github.com/ansible/ansible/issues/65598

Reply:
+1